### PR TITLE
make thing good

### PR DESCRIPTION
**Pull request by 5731la**
fixed double https glitch and made things more humane

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -180,17 +183,19 @@ function upload() {
 			upload=$(curl -s -F "files[]=@"$entry";type=$mimetype" https://api.awau.moe/upload/pomf?key="$key")
 			item="$(egrep -o '"url":\s*"[^"]+"' <<<"${upload}" | cut -d "\"" -f 4)"
 		else
-			echo "ERROR : File size too large or another error occured!"
-			exit 1
```
Comment (5731la) [inline script.sh:184]: useless

File: script.sh [hunk 2]
```diff
@@ -180,17 +183,19 @@ function upload() {
 			upload=$(curl -s -F "files[]=@"$entry";type=$mimetype" https://api.awau.moe/upload/pomf?key="$key")
 			item="$(egrep -o '"url":\s*"[^"]+"' <<<"${upload}" | cut -d "\"" -f 4)"
 		else
-			echo "ERROR : File size too large or another error occured!"
```
Comment (5731la) [inline script.sh:183]: useless

File: script.sh [hunk 3]
```diff
@@ -180,17 +183,19 @@ function upload() {
...
-			exit 1
+			# echo "ERROR : File size too large or another error occured!"
+			#exit 1
+			upload=$(curl -s -F "files[]=@"$entry";type=$mimetype" https://api.awau.moe/upload/pomf?key="$key")
```
Comment (5731la) [inline script.sh:188]: fixed

File: script.sh [hunk 4]
```diff
@@ -180,17 +183,19 @@ function upload() {
...
+                        item="$(egrep -o '"url":\s*"[^"]+"' <<<"${upload}" | cut -d "\"" -f 4)"
```
Comment (5731la) [inline script.sh:189]: fixed

File: script.sh [hunk 5]
```diff
@@ -180,17 +183,19 @@ function upload() {
...
 		fi
 
 	if egrep -q '"success":\s*true' <<< "${upload}"; then
 		d=$2
 		if [ "$d" = true ]; then
 				clipboard "https://$output_url/$item"
-			echo "https://$output_url/$item"
```
Comment (5731la) [inline script.sh:191]: thats the https dupe, the `$output_url` already has https

**Comment by islacantwin** [comment]
nope, what you removed wasnt useless. closing

**Review by 5731la — COMMENTED** [review]
Fixed stuff, read induvidual comments fore more details

**Pull request closed without merging**